Support Android and UWP path configuration

tuket opened the pull request.

jcant0n reviewed (changes requested):
If you try to compile the project after applying the changes in this PR it doesn`t work because the ManualConfig class hasn't implemented the new members in IConfig.

How did you test the changes?

tuket commented:
I have added the missing fields to ManualConfig now.
Also, I have added default library file names for all the platforms.
I have tested it building a local Nuget for Draco.Net and used it from XrvSamples.
